include/criterion/internal/deprecation.h

```diff
@@ -32,4 +32,12 @@
 #  define CR_DEPRECATED_(Msg) _Pragma(#Msg)
 # endif
 
+# ifdef __GNUC__
+#   define CR_DEPRECATED_MEMBER(Member) Member __attribute__((deprecated))
+# elif defined (_MSC_VER)
+#   define CR_DEPRECATED_MEMBER(Member) __pragma deprecated (Member)
```

Comment:
This still doesn't declare the field on msvc -- you need both the pragma & the field definition. Also, for this case, `CR_DEPRECATED_MEMBER(type foo)` fails as the `deprecated` pragma takes an indentifier, not a definition.